src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala

```diff
@@ -24,7 +24,7 @@ class FetchArticlesController(fetchSavedArticles: FetchSavedArticles)(implicit e
            case e: IdentityServiceException => Future.successful( identityErrorResponse )
            case e: MissingAccessTokenException => Future.successful( missingAccessTokenResponse )
            case e: UserNotFoundException => Future.successful(missingUserResponse)
-           case _ => Future.successful(serverErrorResponse )
+           case _ => Future.successful(serverErrorResponse(ex.getMessage) )
```

Comment:
I'm not comfortable with returning any third party exception messages, as you don't always know what the message will include.
Is is safer to log with a UUID and return the UUID?

Reply:
Seen with Nathaniel: this is fixing a crash in prod so let's merge first and fix as a following PR